adafruit_wiznet5k/adafruit_wiznet5k.py

```diff
@@ -336,6 +336,15 @@ def ip_address(self) -> bytes:
         """
         return self._read(_REG_SIPR[self._chip_type], 0x00, 4)
 
+    @property
+    def ipv4_address(self) -> bytes:
```

Comment:
should this show return is string?

Reply:
Fixed! 75ee080404c6218667dd8b320992d284ecb114b4